### PR TITLE
feat: [#23] Lazy loading of occasionally needed code

### LINKED ISSUE #23: Lazy loading of occasionally needed code
Loading JavaScript is expensive. Definitions needed for intermittently used features like the REPL should not be read unless that feature is being used in a particular call.
```
$ time nbb -e '(+ 1 2 3)'
6
nbb -e '(+ 1 2 3)'   0.18s  user 0.02s system 109% cpu 0.185 total
```
```
$ time nbb -e "(require '[clojure.pprint :as pprint])"
nbb -e "(require '[clojure.pprint :as pprint])"   0.21s  user 0.02s system 107% cpu 0.213 total
```

### PR DESCRIPTION
**Pull request by borkdude**
Resolves #23.

### CODE REVIEW

**Comment by borkdude** [comment]
@zane Ready for review.

**Comment by borkdude** [comment]
This PR prepares the brew package for multiple files: https://github.com/babashka/homebrew-brew/pull/1/files

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,7 +18,12 @@ jobs:
       - name: Setup Babashka
         uses: turtlequeue/setup-babashka@v1.3.0
         with:
-          babashka-version: 0.7.0
+          babashka-version: 0.7.3
+
+      - name: Register bb clojure as clojure command
+        run: |
+          echo -e '#!/usr/bin/env bash\n\nbb --clojure "$@"' > /usr/local/bin/clojure
+          chmod +x /usr/local/bin/clojure
```
Comment (zane) [inline .github/workflows/ci.yml:26]: This seems like it's about `nbb --clojure`. I assume it doesn't belong here?

Comment (borkdude) [reply .github/workflows/ci.yml:26]: This is necessary for shadow-cljs which invokes `clojure` for invoking itself. But we already have bb installed which has a clojure runner: `bb clojure`. The n is a newline.

Comment (zane) [reply .github/workflows/ci.yml:26]: Ah! Misread.

**Review by zane — APPROVED** [review]

**Pull request merged**